src/world/area_sbk/sbk_02/92A9A0.c

```diff
@@ -39,16 +39,16 @@ API_CALLABLE(func_802409A8_92AC58) {
     Bytecode* args = script->ptrReadPos;
     s32* buf = (s32*) evt_get_variable(script, *args++);
     s32 i;
-    
+
     if (buf != NULL) {
         for (i = 0; *buf != NULL; i++) {
-            D_80245630[i] = *buf++;
+            sbk_02_D_80245630[i] = *buf++;
         }
-        D_80245630[i] = 0;
+        sbk_02_D_80245630[i] = 0;
     } else {
         for (i = 0; i <= 90; i++) {
-            D_80245630[i] = i + 0x80;
-            D_80245630[91] = 0;
+            sbk_02_D_80245630[i] = i + 0x80;
```

Comment:
these could (or should?) all be converted to #include "world/common/atomic/MakeKeyChoice.inc.c"

Reply:
yeah I figured this would be done in a later pass when the maps were actually tackled, since they'll be re-done anyway